### PR TITLE
Feature/fix trained yolov10 load

### PR DESCRIPTION
**Pull request by HonzaCuhel**
This PR includes:

- Fix for the trained YOLOv10 models loading issue
- Updated version detector for trained YOLOv10 models
- Updated Luxonis logo (the old isn't available anymore)
- Rewritten node.js installation in the Dockerfile

### CODE REVIEW

File: yolo/export_yolov10.py [hunk 1]
```diff
@@ -3,19 +3,122 @@
...
+    it catches the error, logs a warning message, and attempts to install the missing module via the
+    check_requirements() function. After installation, the function again attempts to load the model using torch.load().
+
+    Args:
+        weight (str): The file path of the PyTorch model.
+
+    Returns:
+        (dict): The loaded PyTorch model.
+    """
+    from ultralytics.utils.downloads import attempt_download_asset
```
Comment (conorsim) [inline yolo/export_yolov10.py:34]: Should this import be at the beginning of the file or is there a reason it's deferred?

File: yolo/export_yolov10.py [hunk 2]
```diff
@@ -3,19 +3,122 @@
...
+                )
+            ) from e
+        LOGGER.warning(
+            f"WARNING ⚠️ {weight} appears to require '{e.name}', which is not in ultralytics requirements."
+            f"\nAutoInstall will run now for '{e.name}' but this feature will be removed in the future."
+            f"\nRecommend fixes are to train a new model using the latest 'ultralytics' package or to "
+            f"run a command with an official YOLOv8 model, i.e. 'yolo predict model=yolov8n.pt'"
+        )
+        check_requirements(e.name)  # install missing module
+        ckpt = torch.load(file, map_location="cpu")
```
Comment (conorsim) [inline yolo/export_yolov10.py:71]: Will this second load always work? Could multiple modules be missing?

**Review by conorsim — APPROVED** [review]
Looks good, but I don't have a ton of domain knowledge here. Left a couple of questions.

File: yolo/export_yolov10.py (hunk 1 above)
Comment (HonzaCuhel) [reply yolo/export_yolov10.py:34]: No reason, will move it to the beginning of the file.

File: yolo/export_yolov10.py (hunk 2 above)
Comment (HonzaCuhel) [reply yolo/export_yolov10.py:71]: This function is taken from the Ultralytics package and only updated to fix the issue, so the second load should work.

**Pull request merged**